navit/maptool/osm.c

```diff
@@ -597,6 +602,7 @@ static char *attrmap= {
     "w	boundary=civil		border_civil\n"
     "w	boundary=national_park	border_national_park\n"
     "w	boundary=political	border_political\n"
+    "?	boundary=low_emission_zone	low_emission_zone\n"
```

Comment:
May be "w" in reality, as according to OSM wiki, boundary=low_emmision_zone is only allowed for areas and multi polygon relations which we map to polygons as well. This will result in quite big polygons on map probably. But it is very interesting information.

Reply:
I have not finished working on the low emission zones and this is another PR which should be removed from truck navigation. They are not showing up in the map. Maybe you have an idea why. I use

<itemgra item_types="low_emission_zone" order="0-">
    <polyline color="#00FF00" width="10" dash="5,5"/>
</itemgra>

in layout name="polygons". Changing to "w" type doesn't solve it.

Reply:
I've committed the fix for low_emission_zone. Now you can have it in itemgra. However the combination of width != 1 and dash results in "funny" behaviour as graphics engine cannot dash wide lines due to the way they are drawn. I'd suggest transparent solid line.
To use it for routing however is a different story.

Reply:
Thank you, but it's still not showing up on the map.

Reply:
OK, got it. low emission zones are mostly tagged on boundary relations, not on multipolygon ones. OSM wiki is not clear about this. Wil push fix. Caution: fix renames low_emission_zone to more correct poly_low_emission_zone. Adapt your layout accordingly

Reply:
Perfect. It works fine now. Thank you.